src/docs/product/performance/index.mdx

```diff
@@ -7,36 +7,36 @@ redirect_from:
+The **Performance** page is the main view in [sentry.io](https://sentry.io) where you can search or browse for transaction data. A transaction represents a single instance of an activity you want to measure or track, such as a page load, page navigation, or an asynchronous task. The page displays graphs that visualize transactions or [trends](/product/performance/trends/), as well as a table where you can view relevant transactions and drill down to get more information about them.
 
 ![Performance homepage with All Transactions tab selected.](01_full_widget_all.png)
 
-Using the information on this page, you can trace issues back through services (for instance, frontend to backend) to identify poorly performing code. This allows you to answer questions like whether your application performance is getting better or worse, if your last release is running more slowly than previous ones, or which specific services are slow. Then you can begin to address the code that’s degrading performance.
+Using the information on this page, you can trace issues back through services (for instance, frontend to backend) to identify poorly performing code. You'll be able to determine whether your application performance is getting better or worse, see if your last release is running more slowly than previous ones, and identify specific services that are slow. Once you've found the cause of the problem, you'll be able to address the specific code that’s degrading performance.
 
 ## Filter Performance Data
 
-The <SandboxLink scenario="performance" projectSlug="react">Performance</SandboxLink> page provides you with several filter and display options so that you can focus on the performance data you want. Search conditions help you find the <SandboxLink scenario="oneTransaction" projectSlug="react">transactions</SandboxLink> you want to investigate. You can also control what kind of information is visualized in the [widgets](/product/performance/filters-display/#widgets) or which transactions appear first in the [transactions table](/product/performance/filters-display/#transaction-table).
+The **Performance** page provides you with several filter and display options so that you can focus on the performance data that's most important to you. You can search to find the specific transactions you want to investigate and use [widgets,](/product/performance/filters-display/#widgets) to customize the information being displayed, including which transactions appear first in the [transactions table](/product/performance/filters-display/#transaction-table).
```

Comment:
```suggestion
The **Performance** page provides you with several filter and display options so that you can focus on the performance data that's most important to you. You can use the project, environment, and date filters to customize the information displayed on the page, including what's shown in the [widgets](/product/performance/filters-display/#widgets) and [transactions table](/product/performance/filters-display/#transaction-table). You can also search to find and filter for the specific transactions you want to investigate.
```

This wording makes it sound like widgets are used to customize the info displayed, which isn't true. It's actually the filters that customize the info shown in the widgets. Edited this to make it more accurate